learn/faq.md

```diff
@@ -78,4 +78,6 @@ If you need help with any aspect of the conda ecosystem, feel free to reach out
 
 For more information, read [How to reach us](/community/#how-to-reach-us) on the [Community](/community) page.
 
- 
+## How can I start using conda?
+
+There are several ways to get conda on your computer. You can refer to the [installation guide](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) for a step by step procedure. 
```

Comment:
This is a good start. As a user of conda, do you think you would want more out of an answer like this? Maybe point them to channel or package resources? Actually, a lot of the other FAQ answers cover things like this, so maybe that's unnecessary. We could point to those questions, though. And to help with that, let's put this question right after "What is a feedstock?"

Reply:
Sure, yes. I noticed that we have many questions covering important things I would need as a new user hence didn't add much detail. I'll put this question under "What is feedstock?", see if I can add anything that won't overlap much with another answer and link answers wherever relevant. Thanks!

Reply:
Hi @kathatherine, please take a look at the changes I have made and let me know if this is what you were expecting. Thank you!